wandb/sdk/internal/incremental_table_util.py

```diff
@@ -0,0 +1,25 @@
+import time
+from typing import TYPE_CHECKING
+
+import wandb
+
+if TYPE_CHECKING:
+    from wandb import Table
+
+    from ..wandb_run import Run as LocalRun
+
+ART_TYPE = "wandb-run-incremental-table"
+
+
+def init_artifact(run: "LocalRun", sanitized_key: str):
+    artifact_name = f"run-{run.id}-incr-{sanitized_key}"
+    artifact = wandb.Artifact(
+        artifact_name, "placeholder-run-incremental-table", incremental=True
```

Comment:
nit: if you add a trailing comma and reformat, you get one argument per line, which is generally more readable in these situations

```python
artifact = wandb.Artifact(
    artifact_name,
    "placeholder-run-incremental-table",
    incremental=True,
)
```

same holds for method and function signatures when they don't fit on one line